spec/lib/efile/nj/nj_retirement_income_helper_spec.rb

```diff
@@ -45,98 +154,164 @@
         end
         it "returns #{test_case[:expected]}" do
           helper = Efile::Nj::NjRetirementIncomeHelper.new(intake)
-          result = helper.calculate_maximum_exclusion(test_case[:total_income])
+          result = helper.calculate_maximum_exclusion(test_case[:total_income], test_case[:total_income])
           expect(result).to eq(test_case[:expected])
         end
       end
     end
+  end
 
-    describe ".retirement_exclusion_eligible?" do
-      [
-        { traits: [:mfj_spouse_over_62], line_15: 3_000, line_27: 150_000, line_28a: 37_500, expected: true },
-        { traits: [:primary_over_62], line_15: 3_000, line_27: 150_000, line_28a: 28_125, expected: true },
-        { traits: [:primary_under_62], line_15: 3_000, line_27: 150_000, line_28a: 28_125, expected: false },
-        { traits: [:primary_over_62], line_15: 3_000, line_27: 150_001, line_28a: 0, expected: false },
-        { traits: [:primary_over_62], line_15: 3_001, line_27: 99_999, line_28a: 0, expected: false },
-        { traits: [:primary_over_62], line_15: 3_000, line_27: 150_000, line_28a: 100_000, expected: false },
-      ].each do |test_case|
-        context "when filing with #{test_case}" do
-          let(:intake) do
-            create(:state_file_nj_intake, *test_case[:traits])
-          end
-          let(:instance) do
-            Efile::Nj::Nj1040Calculator.new(
-              year: MultiTenantService.statefile.current_tax_year,
-              intake: intake
-            )
-          end
-          let(:helper) do
-            Efile::Nj::NjRetirementIncomeHelper.new(intake)
-          end
-          before do
-            allow_any_instance_of(Efile::Nj::Nj1040Calculator).to receive(:calculate_line_15).and_return(test_case[:line_15])
-            allow_any_instance_of(Efile::Nj::Nj1040Calculator).to receive(:calculate_line_27).and_return(test_case[:line_27])
-            allow_any_instance_of(Efile::Nj::Nj1040Calculator).to receive(:calculate_line_28a).and_return(test_case[:line_28a])
-            instance.calculate
-          end
-
-          it "returns #{test_case[:expected]}" do
-            result = helper.retirement_exclusion_eligible?(test_case[:line_15], test_case[:line_27], test_case[:line_28a])
-            expect(result).to eq(test_case[:expected])
-          end
+  describe "#line_28b_eligible?" do
```

Comment:
This diff got really messed up because there is both added code and also I fixed indentation on the existing tests. I did not update this describe block